apps/glusterfs/volume_entry.go

```diff
@@ -1010,3 +1010,85 @@ func (v *VolumeEntry) cloneVolumeRequest(db wdb.RODB, clonename string) (*execut
 
 	return vcr, sshhost, nil
 }
+
+// consistencyCheck ... verifies that a volumeEntry is consistent with rest of the database.
+// It is a method on volumeEntry and needs rest of the database as its input.
+func (v *VolumeEntry) consistencyCheck(db Db) (consistent bool, inconsistencies []string) {
```

Comment:
I would have perferred that the consistencyCheck functions be functions rather than methods, but it's not a big deal.
What I don't understand is the need for a seperate bool to indicate an inconsistency when we can check check if the slice is zero length. Doing it with two params feels a bit non-DRY to me.

Reply:
Removed the bool but we still have them as method.